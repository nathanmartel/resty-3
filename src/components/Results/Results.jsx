import React from 'react';
import ReactJson from 'react-json-view';
import PropTypes from 'prop-types';

const Results = ({ item }) => {
  return (
    <>
      <ReactJson src={item} name={false} displayDataTypes={false} />);
      {/* <pre>
        { JSON.stringify(item) }
      </pre> */}
    </>
  );
};

Results.propTypes = {
  item: PropTypes.object.isRequired
};

export default Results;
