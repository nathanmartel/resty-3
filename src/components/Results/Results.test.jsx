import React from 'react';
import { shallow } from 'enzyme';
import Results from './Results';

describe('Results component', () => {
  it('renders Results', () => {
    const item = {
      test: 'Lorem ipsum'
    };
    const wrapper = shallow(<Results item={item} />);
    expect(wrapper).toMatchSnapshot();
  });
});
