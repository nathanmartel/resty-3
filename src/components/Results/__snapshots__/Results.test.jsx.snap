// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results component renders Results 1`] = `
<Fragment>
  <t
    collapseStringsAfterLength={false}
    collapsed={false}
    defaultValue={null}
    displayDataTypes={false}
    displayObjectSize={true}
    enableClipboard={true}
    groupArraysAfterLength={100}
    iconStyle="triangle"
    indentWidth={4}
    name={false}
    onAdd={false}
    onDelete={false}
    onEdit={false}
    onSelect={false}
    shouldCollapse={false}
    sortKeys={false}
    src={
      Object {
        "test": "Lorem ipsum",
      }
    }
    style={Object {}}
    theme="rjv-default"
    validationMessage="Validation Error"
  />
  );
</Fragment>
`;
